### PR TITLE
Card full-bleed images, Statistic recipe + improve MCP guidance for gridliens

### PR DESCRIPTION
**Pull request by rezrah**
## Summary

Towards https://github.com/github/marketing-engineering/issues/5896
Towards https://github.com/github/marketing-engineering/issues/5974
Towards https://github.com/github/brand-experience/issues/287

Adds three things:

- A new `padding` prop to `Card.Image` to allow "full bleed" images like you can with River and Hero.
- A statistic recipe to help agents and downstream users know how to use Statistic with gridlines.
- Updated guidance on gridlines as agents are still getting confused.

## List of notable changes:

- Added `padding` prop to `Card.Image`. Handled all edge cases and preserved existing behavior.
- Updated MCP tool guidance so gridlines are enforced better
- Forwarded `*.example.stories` files through the MCP catalog, which is probably why the above wasn't happening 
- Added a new statistic example

## What should reviewers focus on?

- Check the Card example is correctly implemented
- Same with Statistic

⚠️ There are edge cases where the full bleed card can be used without borders, and inverted. Review those examples too. ⚠️ 

## Steps to test:

1. [Review Cards with full width images](https://primer-888a51e3f5-26139705.drafts.github.io/brand/storybook/?path=/story/components-card-examples--native-borders)
2. [Review the same cards without borders, as this is a valid combination. ](https://primer-888a51e3f5-26139705.drafts.github.io/brand/storybook/?path=/story/components-card-features--image-no-padding-no-borders)
1. [Review statistics with gridlines](https://primer-888a51e3f5-26139705.drafts.github.io/brand/storybook/?path=/story/components-statistic-examples--gridline)

## Contributor checklist:

- [ ] All new and existing CI checks pass
- [ ] Tests prove that the feature works and covers both happy and unhappy paths
- [ ] Any drop in coverage, breaking changes or regressions have been documented above
- [ ] UI Changes contain new visual snapshots (generated by adding `update snapshots` label to the PR)
- [ ] All developer debugging and non-functional logging has been removed
- [ ] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

## Screenshots:

> Please try to provide before and after screenshots or videos

Card example showing full bleed images:

Screenshot 2026-08-04 at 12 20 38

... and without borders (agents have specific guidance not to use this however)

Screenshot 2026-08-04 at 12 25 05

Statistic with gridlines example:
Screenshot 2026-08-04 at 12 25 55

### CODE REVIEW

File: packages/react/src/Card/Card.module.css [hunk 1]
```diff
@@ -256,6 +256,62 @@
   grid-area: image;
 }
 
+.Card__image--padding-none {
+  margin-inline: calc(var(--base-size-32) * -1);
```
Comment (danielguillan) [inline packages/react/src/Card/Card.module.css]: Image

I think this line is causing the image to overflow on narrower viewports because the Card has `20px` padding instead of `32px`. Same thing could happen with the rest where we use `32px` as well. 

Maybe setting the padding in a variable and using it here can make this less error-prone and easier to maintain in the long term.

Comment (rezrah) [reply packages/react/src/Card/Card.module.css]: Nice spot, thanks Dani.

Fixed. Added a dedicated story for it too.

image

**Review by danielguillan — APPROVED** [review]
Looking great!
